elyra/pipeline/tests/test_airlfow_processor.py

```diff
@@ -0,0 +1,237 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import github
+import hashlib
+import mock
+import os
+import pytest
+import tempfile
+
+from elyra.pipeline.processor_airflow import AirflowPipelineProcessor
+from elyra.pipeline.parser import PipelineParser
+from elyra.pipeline.tests.test_pipeline_parser import _read_pipeline_resource
+from elyra.metadata.metadata import Metadata
+from elyra.util import git
+from pathlib import Path
+
+PIPELINE_FILE = 'resources/sample_pipelines/pipeline_dependency_complex.json'
+
+@pytest.fixture
+def processor():
+    processor = AirflowPipelineProcessor(os.getcwd())
+    return processor
+
+
+@pytest.fixture
+def parsed_pipeline():
+    pipeline_resource = _read_pipeline_resource(PIPELINE_FILE)
+    return PipelineParser().parse(pipeline_definitions=pipeline_resource)
+
+
+@pytest.fixture
+def test_metadata():
```

Comment:
I don't know if this is problematic, but I think we should avoid prefixing fixture names with `test_` since those methods _are_ the tests and part of the collection process (with fixtures occurring first I guess).  Perhaps `af_metadata` or `airflow_metadata`.